### DISCUSSION TITLE
service mode

### DISCUSSION THREAD

**Discussion by iccor56** [abraunegg/onedrive#2168] [Q&A] [answered]
if i start onedrive as a service will it monitor changes in the background and update files accordingly or will just be more "ready" to do stuff?

meaning if i use the service do i need to not worry about manually running the onedrive --sync option anymore

**Comment by abraunegg** [comment, marked as answer]
@iccor56 
> if i start onedrive as a service will it monitor changes in the background and update files accordingly

correct - the application will be running in the background, operating as per normal, syncing your data to OneDrive, and pulling down any changes from OneDrive.

> meaning if i use the service do i need to not worry about manually running the onedrive --sync option anymore

Correct

**Reply by iccor56** [reply]
thank you kindly
